src/App.tsx

```diff
@@ -1,23 +1,66 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList/PostList';
+import { Posts } from './types/Post';
+
+export interface Todo {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+}
+
+export interface Comment {
+  postId: number,
+  id: number,
+  name: string,
+  email: string,
+  body: string,
+}
+
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+function getComment(postId: number): Comment[] | null {
+  const foundComment
+  = commentsFromServer.filter((comment) => {
+    return comment.postId === postId;
+  });
+
+  // if there is no user with a given userId
```

Comment:
This comment is incorrect (just duplicated the previous).
I recommend don't leave any comments for now